package.json

```diff
@@ -30,12 +30,12 @@
     "node-fetch": "^2.6.0",
     "pm2": "^4.4.0",
     "pm2-promise": "^2.0.1",
-    "rainbow-bridge-lib": "near/rainbow-bridge-lib#8e5567c0277d13430f952a19bb5e16ceb0b4392c",
+    "rainbow-bridge-lib": "near/rainbow-bridge-lib#373e9f787e51703f186e83e8036dc35131742c1b",
```

Comment:
I see, this is a good way to test rainbow-bridge-lib on ci. There's another way i used:
set `PATCH_RAINBOW_BRIDGE_LIB=373e9f787e51703f186e83e8036dc35131742c1b` env var and run a manual build on rainbow-bridge-cli pipeline on buildkite